pkgs/applications/version-management/gogs/default.nix.patch

```diff
@@ -0,0 +1,22 @@
+2,3c2
```

Comment:
I agree that generated defaults should be easily overridable. I'll look at haskell stuff and try to make it better.